src/repository/opamRepository.ml

```diff
@@ -294,7 +294,14 @@ let pull_tree
         let tmp_archive = OpamFilename.(create tmpdir (basename archive)) in
         OpamFilename.move ~src:archive ~dst:tmp_archive;
         extract_archive tmp_archive (OpamUrl.to_string url)
-      | url, Result None -> Done (Result (OpamUrl.to_string url))
+      | url, Result None ->
+        let url =
+          Printf.sprintf "%s%s"
```

Comment:
We don't see the difference indeed. If we can find a less verbose way to notify about subpathes, it's better, it is used in several places

Reply:
proposal:
```
-> retrieved pin1sub2.dev  (git+file://${BASEDIR}/pinnes[b]#master)
-> retrieved pin1sub1sub2.dev  (file://${BASEDIR}/pinnes[a/k])
```
It will have more effect or url, as we need to insert the subpath in the path itself.

Same differentiation issue with `[]` than `()`.